src/utils/loader.js

```diff
@@ -9,7 +9,7 @@ import {assertDefined} from '../utils/assert'
  * @param {string} appPrefix
  * @param {Function} handleBeforeUnload
  */
-export function loadLocalFile(navigate, appPrefix, handleBeforeUnload) {
+export function loadLocalFile(navigate, appPrefix, handleBeforeUnload, skipAutoRemove = false) {
```

Comment:
if that's there for only for the test, then I'd prefix it like testing_skipAutoRemove